e2etest/bgptests/bgp.go

```diff
@@ -344,16 +344,6 @@ var _ = ginkgo.Describe("BGP", func() {
 							return err
 						}
 
-						err = metrics.ValidateCounterValue(metrics.GreaterThan(0), "metallb_bgp_notifications_sent", map[string]string{"peer": addr}, speakerMetrics)
-						if err != nil {
-							return err
-						}
-
-						err = metrics.ValidateOnPrometheus(promPod, fmt.Sprintf(`metallb_bgp_notifications_sent{peer="%s"} >= 1`, addr), metrics.There)
-						if err != nil {
-							return err
-						}
-
```

Comment:
How is it non-deterministic?
When we configure the bgp peers and advertise the ip don't we always get bgp notification sent?
I don't remember seeing flakes here.

Reply:
Before we always had notifications because of this https://github.com/metallb/metallb/issues/1715
With this new layout, we don't have that change anymore, so the notification is not sent in that scenario (while before, it was ALWAYS been sent), but it still sent _sometimes_.